draft-ietf-ace-coap-est-oscore.md

```diff
@@ -172,9 +172,10 @@ The EST client and/or EST server certificate MAY be a (natively signed) CBOR cer
 
 The {{RFC5272}} specification describes proof-of-possession as the ability of a client to prove its possession of a private key which is linked to a certified public key.
 In case of signature key, a proof-of-possession is generated by the client when it signs the PKCS#10 Request during the enrollment phase.
-Connection-based proof-of-possession is OPTIONAL for EST-oscore clients and servers.
+Connection-based proof-of-possession is OPTIONAL for EST-oscore clients and servers, and it is supported when EDHOC is executed prior to enrollment.
+Connection-based proof-of-possession is not supported when pre-shared OSCORE context is used.
 
-When desired the client can use the EDHOC-Exporter API to extract channel-binding information and provide a connection-based proof-of possession.
+When supported, the client can use the EDHOC-Exporter API to extract channel-binding information and provide a connection-based proof-of possession.
```

Comment:
I suggest to replace "When supported" with "When EDHOC is executed prior to enrollment", since you refer specifically to an EDHOC function. Also, s/EDHOC-Exporter/EDHOC_Exporter.

Reply:
fixed